docs/pages/api-reference/connectors/pubsub.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Pub/Sub
+order: 0
+status: published
+---
+### Pub/Sub
+Data connector to Google Pub/Sub messaging service.
+
+#### Project Parameters
+<Expandable title="name" type="str">
+A name to identify the source. The name should be unique across all Fennel sources.
+</Expandable>
+
+<Expandable title="project_id" type="str">
+The project ID of the Google Cloud project containing the Pub/Sub topic
+</Expandable>
+
+<Expandable title="service_account_key" type="str">
+The JSON string containing the credentials for the Service Account to use to access
+Pub/Sub topic. See below for instructions on how to obtain this.
+</Expandable>
+
+:::info
+Fennel supports only Upsert mode CDC with data in JSON format. If you require support
+for schema or CDC data format, please reach out to Fennel support.
+:::
```

Comment:
I don't follow this. What are we trying to say? Can use not send debezium data in pub-sub? Similarly, can use not do `append` mode only for keyless datasets?

Reply:
@nikhilgarg28 there are some limitations with Pubsub inherently around ordered delivery of messages which prevents us from performing exactly-once semantics. We have listed down some of it here - https://coda.io/d/_dmX2EX1n2_s/Pub-Sub-Fennel-Documentation_suRHk

Reply:
That is correct. Additionally, I am rewording to "supports only Append and Upsert mode with CDC" since we can support Append as well similar to other